aequilibrae/project/project_creation.py

```diff
@@ -64,10 +64,12 @@ def remove_triggers(conn: Connection) -> None:
             for qry in cmd.split("\n"):
                 if qry[:2] == '--':
                     continue
+                while '  ' in qry:
```

Comment:
This is more elegantly solved with regular expressions, so we can build in extra whitespace, and case insensitivity into the regex
```python
# you can cmpile the regex outside the for loop outside the for loop
# we can limit this to the start of a line only if appropriate by using carat '^' in the regex like r'^\s+create....
create_drop_regex = re.compile(r'create\s+trigger\s+(\w+)', flags=re.I)

# search and capture trigger name in one step
m = re.search(create_drop_regex, qry)
if m:
    try:
        curr.execute(f'drop trigger if exists {m.group(1).lower()}')
    ....
```

Reply:
I re-read the documentation, it looks like the re mondule caches recent regexes, so you don't need to bother pre-compiling them.

Reply:
Done and tested